Specs/Number/Chinese/NumberModel.json

```diff
@@ -460,8 +427,7 @@
 ,
 {
   "Input": "负10000",
-  "NotSupportedByDesign": "python",
-  "NotSupported": "javascript",
+  "NotSupported": "javascript, python",
```

Comment:
Any reason this specific case is problematic? I forget if we've discussed it for Javascript.

Reply:
Fixed! (based on #362)